src/main/java/com/microsoft/sqlserver/jdbc/FailOverInfo.java

```diff
@@ -121,13 +121,25 @@ synchronized void failoverAdd(SQLServerConnection connection, boolean actualUseF
     private static final long serialVersionUID = 7393779415545731523L;
 
     private final String serverName;
+    private final String domainName;
     private final int port;
     private final String instanceName;
     private final boolean checkLink;
     private final SQLServerConnectionSecurityManager securityManager;
 
-    ServerPortPlaceHolder(String name, int conPort, String instance, boolean fLink) {
-        serverName = name;
+    ServerPortPlaceHolder(String serverName, int conPort, String instance, boolean fLink) {

```

Comment:
The old constructor can be refactored:

```
    ServerPortPlaceHolder(String serverName, int conPort, String instance, boolean fLink) {
        this(serverName, "", conPort, instance, fLink);
    }
```

Reply:
removed change in this file as discussed